packages/renderer/src/Route.svelte

```diff
@@ -74,11 +78,13 @@ $: route.update({
   breadcrumb,
 });
 
+$: request = requestParser && meta ? requestParser(meta) : ({} as Request);
```

Comment:
suggestion(non-blocking): we could do a first PR to migrate Route component to Svelte 5 to avoid adding new Svelte 4 code here